test/e2e/framework/pod/utils.go

```diff
@@ -338,3 +342,21 @@ func IsPodOnCgroupv2Node(f *framework.Framework, pod *v1.Pod) bool {
 	}
 	return len(out) != 0
 }
+
+// TODO: Remove the rounded cpu limit values when https://github.com/opencontainers/runc/issues/4622
+// is fixed.
+func GetCPULimitCgroupExpectations(milliCPULimit int64) []string {
```

Comment:
This might be getting complicated enough that it could benefit from a unit test.

Reply:
added